docker/Dockerfile.nightly

```diff
@@ -10,7 +10,7 @@ COPY docker/scripts/enable_nightly.sh scripts/enable_nightly.sh
 RUN scripts/enable_nightly.sh
 
 RUN apt-get update \
- && apt-get install -y \
+ && apt-get install --no-install-recommends -y \
```

Comment:
Have you verified this works? My recollection is that some of our packages, such as `gz-*-python` and `gz-*-cli` are suggested/recommended packages and my not install if we add `--no-install-recommends`.